src/Components/GroupForm/GroupForm.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import Button from "../Button/Button.jsx";
+import Select from "../Select/Select.jsx";
+import TextInput from "../TextInput/TextInput.jsx";
+
+import { getPackages, createGroup } from "../../utils/api.js";
+import { languages } from "../../utils/constants.js";
+
+import "./GroupForm.scss";
+
+const GroupForm = ({
+  fixedPackage = false,
+  selectedPackage = null,
+  onSubmitCallback = null,
+}) => {
+  const { t } = useTranslation();
+
+  const [name, setName] = useState("");
+  const [languagePackage, setLanguagePackage] = useState(selectedPackage);
+  const [canSubmit, setCanSubmit] = useState(false);
+  const [languagePackages, setLanguagePackages] = useState([]);
+  const [packageItems, setPackageItems] = useState([]);
+
+  const fetchPackages = useCallback(() => {
+    getPackages(true)
+      .then(({ data }) => {
+        setLanguagePackages(data);
+      })
+      .catch(function (err) {});
+  }, []);
+
+  //make api call to add vocab package
+  async function submitHandler() {
```

Comment:
use arrow function (please search thourgh the complete PR if you used normal functions somewhere else.)